inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -690,6 +743,11 @@ layout layout_optimizer::get_expected_layout(layout const& current_layout,
 
     auto output_layout = node.calc_output_layout();
 
+    // bool valid_onednn_post_ops = get_post_ops_count(node) <= 32;
+
+    // TODO: add logic which respects onednn post-ops limitations and choose optimal clDNN formats
+    // only if "supports_immad" isn't active
```

Comment:
We now have supports_immad. Is this comment still valid?

Reply:
I'll delete it.

Reply:
Done.